marlowe-tutorial/doc/using-marlowe.adoc

```diff
@@ -128,7 +127,7 @@ An alternative execution of the contract is given by
 ----
 *Lang...V2> let (TransactionOutput txWarn2 txPay2 state2 con2) = computeTransaction (TransactionInput (SlotInterval 0 0) [IChoice (ChoiceId "choice" "alice") 0, IChoice (ChoiceId "choice" "bob") 1]) state1 con1
 *Lang...V2> con2
-When [Case (Choice (ChoiceId "choice" "carol") [Bound 1 1]) Close,Case (Choice (ChoiceId "choice" "carol") [Bound 0 0]) (Pay (AccountId 0 "alice") (Party "bob") (Constant 450) Close)] 100 Close
+When [Case (Choice (ChoiceId "choice" "carol") [Bound 1 1]) Close,Case (Choice (ChoiceId "choice" "carol") [Bound 0 0]) (Pay (AccountId 0 "alice") (Party "bob") ada (Constant 450) Close)] 100 Close
 *Lang...V2> state2
 State {accounts = fromList [(AccountId 0 "alice",450)], choices = fromList [(ChoiceId "choice" "alice",0),(ChoiceId "choice" "bob",1)], boundValues = fromList [] , minSlot = 0}
```

Comment:
I was surprised that the state displayed was not changed: I'd have expected to see evidence of multi-currency in this case, because we're exposing the details of the representation of the internal state.